kong/api/routes/config.lua

```diff
@@ -35,6 +41,9 @@ return {
         return kong.response.exit(500, { message = "An unexpected error occurred" })
       end
 
+      _reports._format_version = err_or_ver
```

Comment:
I suggest a more descriptive name for this attribute, since the receiving end may be confused (it somewhat lacks of context). Maybe something like `decl_fmt_version`? Also, `_` prefixed values were for booleans (yes/no enabled listeners etc). For this attribute, better removing it.